### ISSUE TITLE
Compatibility Issue

### ISSUE THREAD

**Issue by heaucques** [Orbiit/gunn-remake#12]
Navigation bar's `a` lacks a `color` transitioning effect

**Comment by SheepTester** [comment]
*accessibility; compatibility means it doesn't work on other devices and versions and such

It doesn't need a transitioning effect; especially since the change is so faint, adding an animation will make the change even harder to notice

**Comment by SheepTester** [comment]
I'm assuming this has been resolved...?

**Comment by SheepTester** [comment]
Also, you can add a transition effect yourself, you know
